docs/uml/c4_build.puml

```diff
@@ -0,0 +1,32 @@
+@startuml
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Component.puml
+
```

Comment:
A few `skinparams` I like to use with C4 diagrams:

```puml
skinparam linetype ortho
skinparam nodesep 100
skinparam ranksep 250
scale 1920x1080
```

Diagrams with angled lines look a bit less _messy_, and rank and node separators move elements further apart so that text on the lines is not cut off. Finally, scale ensures consistent images rendering.

Reply:
Nice! I've added these, it does make the diagram look better